### DISCUSSION TITLE
Get deepgram to spell out words in the TTS

### DISCUSSION THREAD

**Discussion by solomon23** [deepgram/community#812] [General help] [answered]
I can't find a good reliable way to get deepgram to spell out words.  i've tried
B R A V V
B.R.A.V.V.
B-R-A-V-V
bee arr ayy ayy vee

etc - anyone have a way to make this work ?

i'm looking for something similiar to 

`<say-as interpret-as="characters" format="characters">BRAVV</say-as>`

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by SandraRodgers** [comment, marked as answer]
For now, you can use these words to represent the letters that you are spelling out:

Eigh, bee, sea, Dee, Eeeee, eff, gee, aitch, eye, Jay, Kay, Elle , Emm, En, Oh, pee, queue, Ar, ess, tee, you, vee, double you, eks, why, zee.

Soon, we hope to have a better solution.
